examples/fabric-bridge-app/fabric-bridge-common/BUILD.gn

```diff
@@ -45,11 +45,9 @@ source_set("fabric-bridge-lib") {
     "include/BridgedDeviceBasicInformationImpl.h",
     "include/BridgedDeviceManager.h",
     "include/CHIPProjectAppConfig.h",
-    "include/CommissionerControl.h",
```

Comment:
It looks like you are moving these files from `examples/fabric-bridge-app/fabric-bridge-common/` to `examples/fabric-bridge-app/linux/`. But this PR add to new location and doesn't remove from previous location. That makes it harder to review the delta.

Reply:
CommissionerControl.h/.cpp is using RPC which make them Linux dependent, removed the old files.